youtube_dl/extractor/pluralsight.py

```diff
@@ -138,6 +139,8 @@ def _real_extract(self, url):
                 format_id = '%s-%s' % (ext, quality)
                 clip_url = self._download_webpage(
                     request, display_id, 'Downloading %s URL' % format_id, fatal=False)
+                if sleep_interval:
+                    self._sleep(sleep_interval, display_id)
```

Comment:
Can you force this sleep to some reasonable value instead of using `sleep_interval` and add a rationale comment?